### ISSUE TITLE
Problem building documentation on RHEL7

### ISSUE THREAD

**Issue by ralphlange** [epics-base/pvaPy#53]
pvaPy 2.0.0
Base (C++) 7.0.3.1
Python 2.7

On RHEL7, the `sphinx_rtd_theme` can be installed from EPEL as [`python2-sphinx_rtd_theme-0.2.4-3.el7.noarch.rpm`.](https://centos.pkgs.org/7/epel-x86_64/python2-sphinx_rtd_theme-0.2.4-3.el7.noarch.rpm.html) (This pulls a few dependencies for the fonts.)

After that, the pvaPy documentation build still fails with:
```
make[1]: Entering directory `/home/DCS/langer/work/CODAC/m-python-pvapy/target/pvaPy-2.0.0/documentation'
. /home/DCS/langer/work/CODAC/m-python-pvapy/target/pvaPy-2.0.0/bin/linux-x86_64/pvapy_setup_full.2.7.sh; make -C sphinx default
make[2]: Entering directory `/home/DCS/langer/work/CODAC/m-python-pvapy/target/pvaPy-2.0.0/documentation/sphinx'
export PATH=`dirname /usr/bin/sphinx-build`:/usr/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/home/DCS/langer/bin && make -f Makefile.sphinx html
make[3]: Entering directory `/home/DCS/langer/work/CODAC/m-python-pvapy/target/pvaPy-2.0.0/documentation/sphinx'
sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v1.1.3
loading pickled environment... not yet created
[ERROR] Theme error:
[WARN] no theme named 'sphinx_rtd_theme' found (missing theme.conf?)
[WARN] make[3]: *** [html] Error 1
make[3]: Leaving directory `/home/DCS/langer/work/CODAC/m-python-pvapy/target/pvaPy-2.0.0/documentation/sphinx'
make[2]: Leaving directory `/home/DCS/langer/work/CODAC/m-python-pvapy/target/pvaPy-2.0.0/documentation/sphinx'
make[1]: Leaving directory `/home/DCS/langer/work/CODAC/m-python-pvapy/target/pvaPy-2.0.0/documentation'
[WARN] make[2]: *** [html] Error 2
[WARN] make[1]: *** [default] Error 2
[WARN] make: *** [doc] Error 2
```

What am I missing?

**Comment by sveseli** [comment]
I typically install sphinx_rtd_theme via pip, which seems to pull version 0.4.3:

<pre>
bluegill2> pip install sphinx_rtd_theme
Collecting sphinx_rtd_theme
  Downloading https://files.pythonhosted.org/packages/60/b4/4df37087a1d36755e3a3bfd2a30263f358d2dea21938240fa02313d45f51/sphinx_rtd_theme-0.4.3-py2.py3-none-any
...
Installing collected packages: sphinx-rtd-theme
Successfully installed sphinx-rtd-theme-0.4.3

bluegill2> python -c "import sphinx_rtd_theme"
bluegill2> 
</pre>

The installed package has theme.conf file:

<pre>
bluegill2> ls -l /local/sveseli/CTLSDAQ2/support-7.0-svn/opt/python-2.7.16/linux-x86_64/lib/python2.7/site-packages/sphinx_rtd_theme/theme.conf
-rw-r--r-- 1 sveseli xsdsdm 355 Nov 18 09:46 /local/sveseli/CTLSDAQ2/support-7.0-svn/opt/python-2.7.16/linux-x86_64/lib/python2.7/site-packages/sphinx_rtd_theme/theme.conf
</pre>

I notice that you are installing an older version of sphinx_rtd_theme, which does not seem to have theme.conf file. Could you try installing this via pip rather than from EPEL? If this is a problem, I can probably come up with a scheme to avoid using rtd theme for the doc build.

**Comment by ralphlange** [comment]
Back on this...
Our building process is not allowed to have internet access - so pip installs are out of the question.
I will investigate how and why the official EPEL install differs ...

**Comment by ralphlange** [comment]
Well, 
```
$ python -c "import sphinx_rtd_theme"
$ 
```
works like in your case. Also, the configuration is there, in a location that looks appropriate:
```
$ ls -l /usr/lib/python2.7/site-packages/sphinx_rtd_theme/theme.conf
-rw-r--r-- 1 root root 260 Mar  6  2017 /usr/lib/python2.7/site-packages/sphinx_rtd_theme/theme.conf
```

**Comment by sveseli** [comment]
I reproduced the problem using sphinx 1.1.2. Could you try building in your environment using the doc_fix branch, and if that works, I'll prepare a new release.

**Comment by ralphlange** [comment]
Switching to the 'default' theme works, ack.

However, older Sphinx documentation (https://www.sphinx-doc.org/en/1.8/theming.html) says the rtd theme was introduced in 1.3, so making the switch at version >= 13 would seem more appropriate. Obviously, that doesn't affect my "11" (1.1.3) version.

**Comment by sveseli** [comment]
This should be fixed as of release 2.1.0.
